lib/promenade/karafka/statistics_subscriber.rb

```diff
@@ -0,0 +1,81 @@
+require "promenade/karafka/subscriber"
+require "active_support/core_ext/hash"
+
+module Promenade
+  module Karafka
+    class StatisticsSubscriber < Subscriber
+      attach_to "statistics.karafka"
+
+      Promenade.histogram :kafka_connection_latency do
+        doc "Request latency (rtt)"
+        buckets :network
+      end
+
+      Promenade.counter :kafka_connection_calls do
+        doc "Count of calls made to Kafka broker"
+      end
+
+      Promenade.gauge :kafka_consumer_ofset_lag do
+        doc "Lag between message create and consume time"
+      end
+
+      def emitted(event)
+        group = event.payload[:consumer_group_id]
+        statistics = event.payload[:statistics].with_indifferent_access
+
+        report_topic_metrics(statistics, group)
+        report_connection_metrics(statistics)
+      end
+
+      private
+
+        def report_topic_metrics(statistics, group)
+          statistics[:topics].map do |topic_name, topic_values|
```

Comment:
nitpick looks like this method only needs to know about topics array:
```suggestion
        def report_topic_metrics(topics, group)
          topics.map do |topic_name, topic_values|
```

```ruby
report_topic_metrics(statistics[:topics], group)
```